src/main/api/studio-api.yaml

```diff
@@ -9369,17 +9369,13 @@ components:
             type: object
             properties:
                 item:
-                    $ref: '#/components/schemas/ContentItem'
+                    $ref: '#/components/schemas/ContentItemV1'
```

Comment:
What's this?

Reply:
That API (/api/2/configuration/get_configuration_history) spec was not matching its actual response.

Reply:
Will we have two return types, ContentItem and ContentItemV1?

Reply:
I will update `get_configuration_history` to return the new one. 
Notice ContentItemV1 is still returned by some V1 APIs. So it can be removed once those APIs are retired:
- /api/1/services/api/1/content/get-item.json
- /api/1/services/api/1/content/get-item-orders.json
- /api/1/services/api/1/content/get-pages.json
- /api/1/services/api/1/content/get-items-tree.json